cinema/serializers.py

```diff
@@ -1 +1,72 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Actor, Genre, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
```

Comment:
Ihor suggested to leave a blank line between main class and a sub class)

Reply:
ty, will fix

Reply:
@aLEKS-e3 10 points to Gryffindor!